### DISCUSSION TITLE
Event propagation delay

### DISCUSSION THREAD

**Discussion by matejvasek** [containers/podman#11204] [General]
Hello all,

I am trying to run `podman` in katacoda environment.
https://learn.openshift.com/developing-on-openshift/serverless/

I am running `podman` service in one VM and accessing ti via `ssh://` from another VM.

When a container exists the `died` event should be propagated.
The issue is that the event is not fired immediately when it should.
The event is however fired when I do some other request to the `podman` service (e.g. `curl -X GET --unix-socket /tmp/podman.sock "http://d/v1.40/info"`).

Any idea what could be causing it?

**Comment by mheon** [comment]
I suspect that the VM might be pausing while no user interaction is occuring (to try and conserve resources, I'd imagine?) and that is causing the Podman cleanup process to not run until further user interaction occurs? I'm not a Katacoda expert, but the fact that we haven't seen this outside of that environment makes me suspect Katacoda and not Podman.
